content/zh-cn/docs/concepts/security/_index.md

```diff
@@ -2,4 +2,253 @@
 title: "安全"
 weight: 85
 description: 确保云原生工作负载安全的一组概念。
+simple_list: true
 ---
+
+<!--
+title: "Security"
+weight: 85
+description: >
+  Concepts for keeping your cloud-native workload secure.
+simple_list: true
+-->
+
+<!--
+This section of the Kubernetes documentation aims to help you learn to run
+workloads more securely, and about the essential aspects of keeping a
+Kubernetes cluster secure.
+
+Kubernetes is based on a cloud-native architecture, and draws on advice from the
+{{< glossary_tooltip text="CNCF" term_id="cncf" >}} about good practice for
+cloud native information security.
+-->
+Kubernetes 文档的这一部分内容的旨在引导你学习如何更安全地运行工作负载，
+以及维护 Kubernetes 集群的基本安全性。
+
+Kubernetes 基于云原生架构，并借鉴了
+{{<lossary_tooltip text="CNCF" term_id="cncf" >}} 有关云原生信息安全良好实践的建议。
```

Comment:
Typo in the `glossary_tooltip` shortcode name is leading to [build failures](https://app.netlify.com/sites/kubernetes-io-main-staging/deploys/65d0b4b1962ad40008cab067).

```suggestion
{{< glossary_tooltip text="CNCF" term_id="cncf" >}} 有关云原生信息安全良好实践的建议。
```